src/Components/Artwork/__tests__/Details.test.tsx

```diff
@@ -101,20 +119,25 @@ const artworkInAuction = {
   partner: {
     name: "Forum Auctions",
     href: "/auction/forum-auctions",
-    __id: "UGFydG5lcjpmb3J1bS1hdWN0aW9ucw==",
   },
   sale: {
     is_auction: true,
-    is_live_open: false,
-    is_open: true,
     is_closed: false,
     display_timely_at: "ends in 14d",
-    __id: "U2FsZTpmb3J1bS1hdWN0aW9ucy1tdWx0aXBseQ==",
   },
   sale_artwork: {
-    highest_bid: { display: "$2,600", __id: null },
+    highest_bid: { display: "$2,600" },
     opening_bid: { display: "$2,400" },
-    __id: "U2FsZUFydHdvcms6Z2VyaGFyZC1yaWNodGVyLXR1bGlwcy1wMTctMTQ=",
+    counts: { bidder_positions: 0 },
   },
-  __id: "QXJ0d29yazpnZXJoYXJkLXJpY2h0ZXItdHVsaXBzLXAxNy0xNA==",
 }
+
+// So, we make changes to the fixtured data in tests
+// you'll need to use a version that's Any'd so that
+// tsc passes. Trade-off of any in the test vs guarantee
+// of the data in the fixture.
+//
+// Maybe we could use something like a
+// Muatable<Partial<Details_artwork>>
+// instead?
```

Comment:
Nope, that would only work if that fragment has no fragments of child containers spread in. That’s what the first item in this issue is about https://github.com/facebook/relay/issues/2597

Reply:
Thank you sire